app/swagger/schemas/async_transaction/vet360.rb

```diff
@@ -32,6 +32,11 @@ class Vet360
             end
           end
         end
+
+        swagger_schema :AsyncTransactionsVet360 do
+          key :required, [:data]
+          property :data, type: :array
```

Comment:
Aren't we responding with more than just a `data` prop?

Reply:
`data` is an array of 0-3 AsyncTransaction objects. It don't know how to get the swagger DSL to express that.

Reply:
I believe you could declare it as an array of some item type that is an object that contains all the correct field, but  use a string enum when it comes to the type field to show that it could be any of the 3.